react/package.json

```diff
@@ -3,10 +3,13 @@
   "private": true,
   "version": "0.0.0",
   "type": "module",
+  "packageManager": "pnpm@8.1.1",
```

Comment:
Can we specify a range instead of a specific version? e.g. ^8 or ^8.1.0.
Maybe add node.js version as well (>=16)?

Reply:
No, corepack accepts a specified version only :(
We can use renovate bot to bump this automatically